design/baremetal-operator/how-ironic-works.md

```diff
@@ -7,8 +7,10 @@
 
 # how-ironic-works
 
+<!-- cSpell:ignore irmc,xclarity,uncompress -->
```

Comment:
Should we replace uncompress with decompress?

Reply:
Done